package.json

```diff
@@ -1,29 +1,37 @@
 {
   "name": "cbioportal-frontend",
+  "private": true,
+  "version": "0.0.0",
+  "workspaces": {
+    "packages": [".", "packages/*"]
+  },
   "description": "",
   "scripts": {
     "start": "./scripts/env_vars.sh && eval \"$(./scripts/env_vars.sh)\" && cross-env NODE_ENV=development webpack-dev-server",
     "startSSL": "./scripts/env_vars.sh && eval \"$(./scripts/env_vars.sh)\" && cross-env NODE_ENV=development webpack-dev-server --https",
+    "startAll": "lerna run start --parallel",
     "clean": "rimraf dist tsDist common-dist",
     "build": "yarn run clean && yarn run compileOqlParser && yarn run buildDLL:prod && cross-env NODE_ENV=production NODE_OPTIONS=--max-old-space-size=2048 webpack",
+    "buildAll": "lerna run build --stream",
+    "buildModules": "lerna run build --ignore=cbioportal-frontend --stream",
+    "publishModules": "lerna run publish from-package --yes",
```

Comment:
publish packages that are not already published to npm, by checking the `version` of each `package.json` (see https://github.com/lerna/lerna/tree/master/commands/publish#bump-from-package)